kinto/__main__.py

```diff
@@ -93,6 +99,4 @@ def main(args=None):
             pserve_argv.append('--reload')
         pserve.main(pserve_argv)
 
-
-if __name__ == "__main__":
-    main()
```

Comment:
If you use ``pip install -e .`` then it should update when updating the file without having to restart ``pip install`` in between.